improver/calibration/utilities.py

```diff
@@ -272,3 +272,81 @@ def merge_land_and_sea(calibrated_land_only, uncalibrated):
         mask = calibrated_land_only.data.mask
         new_data[mask] = uncalibrated.data[mask]
         calibrated_land_only.data = new_data
+
+
+def forecast_coords_match(first_cube, second_cube):
+    """
+    Determine if two cubes have equivalent forecast_periods and that the hours
+    of the forecast_reference_time coordinates match. Only the points of the
+    forecast period and forecast reference time coordinate are checked. This is
+    to ensure that an EMOS coefficient cube matches the forecast cube, as
+    appropriate.
+
+    Args:
+        first_cube (iris.cube.Cube):
+            First cube to compare.
+        second_cube (iris.cube.Cube):
+            Second cube to compare.
+
+    Raises:
+        ValueError: The two cubes are not equivalent.
+    """
+
+    def _frt_hours(cube):
```

Comment:
Is this needed?  Could you just use the `get_frt_hours` utility below?

Reply:
I've removed this and used the `get_frt_hours` utility instead.